spec/features/users/user_deletes_a_question_spec.rb

```diff
@@ -0,0 +1,24 @@
+require "rails_helper"
+
+feature "user visits the question show page" do
+  scenario "they click delete question" do
+    category = Fabricate(:category, name: "Space")
+    user = Fabricate(:user, name: "Space Nerd")
+    question = Fabricate(:question,
+                                      title: "How high is the ISS?",
+                                      body: "So high.",
+                                      user: user,
+                                      category: category )
+
+    visit question_path(question)
+
+    within("#delete-question") do
+      click_on "Delete Question"
+    end
+
+    expect(current_path).to eq(root_path)
+    expect(page).to have_content("Your question was deleted successfully!")
```

Comment:
I would find a way to affirm that a question path is in fact deleted. If theres a way to expect that visiting question_path(question) would fail I would assume that would be good.